odpf/guardian/v1beta1/guardian.proto

```diff
@@ -448,7 +448,12 @@ message Policy {
         repeated AdditionalAppeal appeals = 2;
     }
     repeated Requirement requirements = 8;
-    google.protobuf.Struct iam = 9;
+
+    message IAM {
+        string provider = 1;
+        google.protobuf.Value config = 2;
```

Comment:
if it is always going to be a map, we could use `Struct` instead of `Value`. wdyt?